### ISSUE TITLE
Midi Through?

### ISSUE THREAD

**Issue by sam-n** [YuuichiAkagawa/USBH_MIDI#16]
Hey there, thank you very much for this great library. It works really well for converting USB to standard MIDI and I have used it already quite a lot. I was just wondering if it is possible to do something like a MIDI through? I have tried it already by sending the message in the outBuf back to the piano within the MIDI_poll() function. That seems to be fine, but after I play a few past notes on my keyboard it starts playing up and I triggers notes that I am actually not playing for some reason. Do you have any suggestions?

Many thanks for your help

Sam

**Comment by monkeybiscuits** [comment]
Yes, being able to daisy chain a USB midi device would be useful. Or being able to use a non-MIDI USB device to trigger MIDI messages to a computer DAW.
